project.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "devcycle-js-sdks",
```

Comment:
what is file this for?

Reply:
its a file that nx keeps automatically adding every time you generate a new package. I was thinking of just giving up and letting it be generated...